packages/lib/checkRateLimitAndThrowError.ts

```diff
@@ -65,6 +72,15 @@ async function changeSMSLockState(identifier: string, status: SMSLockState) {
       },
     });
   } else {
+    const team = await prisma.team.findUnique({
+      where: { id: teamId, parentId: null, isOrganization: false },
+    });
+    if (team?.isSMSLockStateReviewedBefore)
+      throw new TRPCError({
+        code: "CONFLICT",
+        message: "Re-review is not allowed",
+      });
+
```

Comment:
We don't want to throw error. We just don't want to mark the user or team as review needed or locked if it had previously happened and admin then unlocked it.

If it was previously reviewed and state is unlocked, you should just return without updating the state or making any changes to the db.